docs/walkthrough.md

```diff
@@ -156,15 +160,13 @@ spec:
       labels:
         app: prometheus
     spec:
-      serviceAccountName: prom-cm-adapter 
+      serviceAccountName: prom-cm-adapter
       containers:
-      - image: prom/prometheus:v1.6.1
+      - image: prom/prometheus:v2.1.0
```

Comment:
there are some known problems with v2.1.0, v2.2.0-rc.0 is probably more stable

Reply:
ack